documentation/source/intro/overview.rst

```diff
@@ -0,0 +1,90 @@
+.. _Getting-Started:
+
+OpenEBS Introduction
+====================
+OpenEBS is a cloud native storage solution built with the goal of providing containerized storage for containers. Using OpenEBS, a developer can seamlessly get the persistent storage for stateful applications with ease, much of which is automated, while using the popular orchestration platforms such as Kubernetes.
+
+OpenEBS is Container Attached Storage(CAS) - which combines the best of DAS and NAS Storage Solutions.
+
+.. image:: ../_static/das-nas-cas.png
+    :align: center
+
+At the outset, OpenEBS provides you with:
+
+* A *simple to use* storage solution (simpler than using local storage volumes) for your persistent applications, 
+* And includes benefits of external storage like snapshots, backup, compression etc. 
+* Allows you to run Stateful Applications in Kubernetes on any node (any cloud or any on-premise server), any storage (disks attached to servers). Avoid Storage Vendor lock-in!
+
+OpenEBS is Containerized Storage Solution that can be orchestrated by any Container Orchestrators, however - the current versions support orchestration using Kubernetes. To understand how to use OpenEBS with Kubernetes, we highly recommend that you familiarize with `Kubernetes Storage Concepts`_, specifically:
```

Comment:
is a...
Orchestrators. However, the current...
we highly = OpenEBS highly 
recommend = recommends

Reply:
done. with slide modifications.